data/sql_updates/create_totals_presidential_view.sql

```diff
@@ -42,7 +42,8 @@ from
     left join dimdates start_date on cvg_start_dt_sk = start_date.date_sk and cvg_start_dt_sk != 1
     left join dimdates end_date on cvg_end_dt_sk = end_date.date_sk and cvg_end_dt_sk != 1
 where
-    p.expire_date is null or p.expire_date > date_trunc('day', now())
+    (p.expire_date is null or p.expire_date > date_trunc('day', now()))
```

Comment:
@arowla do you have a preference here? Based on @LindsayYoung's description, it sounds like we don't need to check for expired dates that are in the future, and I confirmed that none of the facts tables have expired dates that haven't already happened:
```
fec=> select count(*) from factpresidential_f3p where expire_date > date_trunc('day', now());
 count
-------
     0
```
Any reservations about dropping the `expire_date` part of the query, or should we leave it in?